logging/include/Logging.hpp

```diff
@@ -31,30 +32,36 @@ constexpr LogLevel CURRENT_LEVEL = LogLevel::info;
 
 class Logger {
   std::string _name;
-  std::string LEVELS_STRINGS[6] = {"TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
+  std::array<std::string, 6> LEVELS_STRINGS = {"TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
   std::unordered_map<std::string, std::string> mdc_;
+  mutable std::mutex mdc_mut_;
```

Comment:
Nit: Using a full word like `mdc_mutex_` makes it more apparent that mut means mutex and not mutable. For most of our variables we try to use full words.

Reply:
Thanks @andrewjstone. I've pushed another commit with the mutex renamed.